srv/SetForceMode.srv

```diff
@@ -0,0 +1,21 @@
+# A 6d pose vector that defines the force frame relative to the base frame
+geometry_msgs/PoseStamped task_frame        
+
+# A 6d vector of 0s and 1s. 1 means that the robot will be compliant in the corresponding axis of the task frame
+bool selection_vector_x
+bool selection_vector_y
+bool selection_vector_z
+bool selection_vector_rx
+bool selection_vector_ry
+bool selection_vector_rz  
```

Comment:
I'm not sure whether "selection_vector" is the best naming. Maybe something like "compliance_x" would be more suitable?